EcoSoapBank/Network Controller/GraphQLResult.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  GraphQLResult.swift
+//  EcoSoapBank
+//
+//  Created by Shawn Gee on 9/2/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+/// A container for decoding the result of a GraphQL query or mutation
+struct GraphQLResult<T: Decodable>: Decodable {
```

Comment:
Good call separating this out; makes the decoding a bit more "composable" and compartmentalizes different parts of it.